quicklogic/openfpga/devices/k4_N8/k4_N8_tileable_reset_softadder_register_scan_chain_nonLR_caravel_io_skywater130nm.xml

```diff
@@ -0,0 +1,786 @@
+<!-- 
```

Comment:
It is unclear to me if this XML file should be directly landed here....

Reply:
At least for now QuickLogic wants the VPR arch file to be part of the SymbiFlow repository.